rdt/transformers/numerical.py

```diff
@@ -1,16 +1,58 @@
 """Transformers for numerical data."""
-
 import copy
 
 import numpy as np
 import pandas as pd
 import scipy
-from copulas import EPSILON, univariate
 
 from rdt.transformers.base import BaseTransformer
 from rdt.transformers.null import NullTransformer
 
 
+def _get_distributions():
```

Comment:
Since this is specific to the Transformer class only, I'd put it as a `@staticmethod` inside it.